### PR TITLE
merge in series02 files

### PR DESCRIPTION
**Pull request by Ginsusamurai**
series, fizzbuzz and grid exercises

### CODE REVIEW

File: students/morgan/session02/grid.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+
+corner  = "+ "
```
Comment (PythonCHB) [inline students/morgan/session02/grid.py:3]: I like this!

Did you experiment with putting different characters in there to see how well it worked?

File: students/morgan/session02/series.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+def lucas(n):
+	""" lucas number sequence. same as fib but starts with 2 and 1"""
+	if n == 0:
+		return 2
+	if n == 1:
+		return 1
+	return lucas(n-1) + lucas(n-2)
+
+def sum_series(n, b=0, c=1):
```
Comment (PythonCHB) [inline students/morgan/session02/series.py:17]: Perfect!

File: students/morgan/session02/series.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	print(fib(2))
+
+	assert lucas(0) == 2
+	assert lucas(1) == 1
+	print(lucas(2))
+
+	assert sum_series(0,3,8) == 3
+	assert sum_series(1,3,8) == 8
+	print(sum_series(2,3,8))
+
```
Comment (PythonCHB) [inline students/morgan/session02/series.py:39]: you want to make sure to test a couple larger values, not just the initial ones.

**Review by PythonCHB — APPROVED** [review]
This all looks good.

EXCEPT -- it looks like you are indenting with tabs. You really, really, don't want to do that.

The style standard for python code is to use 4 spaces for indenting -- always. If you are ever going to sahre code with anyone, you really, really want to use that standard.

Set up your editor to put in spaces when you hit the <tab> key, and you should be good to go.

File: students/morgan/session02/grid.py (hunk 1 above)
Comment (Ginsusamurai) [reply students/morgan/session02/grid.py:3]: a bit. it looks fine, just not as even. +, -, | just look more square on the grid.

Comment (PythonCHB) [reply students/morgan/session02/grid.py:3]: yup -- those probably are the best options -- but goodto knowthat the does work with other characters..

**Pull request merged**